### PR TITLE
Add support for custom fields updating person

### PR DESCRIPTION
**Pull request by chemitaxis**
Hi, we need to send custom updated fields, and in this moment, Pipedrive SDK does not allow, please accept this PR ASAP. Thank you so much.

### CODE REVIEW

**Comment by chemitaxis** [comment]
This is very similar PR to https://github.com/pipedrive/client-nodejs/pull/116 created by @RuTsEST

File: lib/Controllers/PersonsController.js [hunk 1]
```diff
@@ -627,15 +627,17 @@ class PersonsController {
             visible_to: (input.visibleTo !== null) ? input.visibleTo : null,
         };
```
Comment (RuTsEST) [inline lib/Controllers/PersonsController.js]: Set these properties as undefined/null so the returned object wouldn't have duplicate properties in both camelCase and snake_case.
```suggestion
		input.ownerId = undefined;
        input.orgId = undefined;
        input.visibleTo = undefined;
```

Comment (chemitaxis) [reply lib/Controllers/PersonsController.js]: @RuTsEST done!!

**Comment by chemitaxis** [comment]
@RuTsEST I have updated the PR, thanks for the feedback!

**Comment by chemitaxis** [comment]
@RuTsEST can you please check it? Thanks!!

File: lib/Controllers/PersonsController.js [hunk 2]
```diff
@@ -627,15 +627,21 @@ class PersonsController {
...
 
+        input.ownerId = undefined;
+        input.orgId = undefined;
+        input.visibleTo = undefined;
+
+        const _formWithDynamicFields = {..._form, ...input};
```
Comment (RuTsEST) [inline lib/Controllers/PersonsController.js]:
```suggestion
        const _formWithDynamicFields = {...input, ..._form}; 
```

Comment (RuTsEST) [reply lib/Controllers/PersonsController.js]: This is so that the _form will not get overridden by the undefined values

**Comment by chemitaxis** [comment]
Done @RuTsEST :)

**Comment by chemitaxis** [comment]
@RuTsEST please ping me when you release a new version, thanks!

**Pull request merged**